moveit_ros/visualization/planning_scene_rviz_plugin/src/planning_scene_display.cpp

```diff
@@ -662,7 +665,8 @@ void PlanningSceneDisplay::calculateOffsetPosition()
   Ogre::Vector3 position;
   Ogre::Quaternion orientation;
 
-  context_->getFrameManager()->getTransform(getRobotModel()->getModelFrame(), rclcpp::Time(0), position, orientation);
+  context_->getFrameManager()->getTransform(getRobotModel()->getModelFrame(), rclcpp::Time(0, 0, RCL_ROS_TIME),
```

Comment:
This will fix the exception of two different time resources comparison

Reply:
Ok